pandas/tests/groupby/methods/test_describe.py

```diff
@@ -219,3 +219,73 @@ def test_describe_duplicate_columns():
     )
     expected.index.names = [1]
     tm.assert_frame_equal(result, expected)
+
+
+class TestGroupByNonCythonPaths:
```

Comment:
Note for a follow up, would be good to collapse this class into just the testing function and fixtures